test/inductor/test_max_autotune.py

```diff
@@ -1438,17 +1439,14 @@ def f(x, y):
     @config.patch(
         max_autotune=True,
         max_autotune_gemm_backends="TRITON",
+        comprehensive_padding=False,
+        shape_padding=False,
+    )
+    @patch_matmul(
```

Comment:
Any reason you can't just reuse `config.patch` here? This feels like more infrastructure to maintain that isn't reused.

Reply:
Okay makes sense. I don't know enough about this part of the codebase to know if we want to unify eventually, but this seems safer for now.